roles/openshift_cluster_monitoring_operator/README.md

```diff
@@ -0,0 +1,33 @@
+# OpenShift Cluster Monitoring Operator
+
+OpenShift Cluster Monitoring Operator manages the TPO deployment.
```

Comment:
For the acronym challenged please use full name for TPO. I'm confused how this relates to prometheus operator.

Reply:
TPO is Tectonic Prometheus Operator, and I'm not sure we want to use the name Tectonic here, it might be confusing from a branding prespective, so I just used "TPO" for now.

You can read about it here: https://coreos.com/tectonic/docs/latest/tectonic-prometheus-operator/tectonic-monitoring.html

The end goal, I assume, is to manage all the cluster monitoring for OpenShift via this operator instead of using Ansible.

Reply:
I think we can just name it "cluster monitoring operator" moving forward. Once we find out where the continued version of the TPO will live we will just rename it along moving it.